components/filter/README.md

```diff
@@ -27,6 +27,7 @@ The `d2l-filter` component allows a user to filter on one or more dimensions of
 
 **Events:**
 * `d2l-filter-change`: dispatched when any filter value has changed (may contain info about multiple changes)
+* `d2l-filter-dimension-open`: dispatched when a dimension is opened (if there is only one dimension, this will be dispatched when the dropdown is opened)
```

Comment:
Wondering if `d2l-filter-dimension-first-open` would be easier for folks to use in a lazy load scenario? Otherwise they might have to track this themselves or remove their event listener.

Reply:
I could!  For the labs component, they currently just listen for the "tab-selected" event which is dispatched every time, so they're used to having to handle it.  But this would be a nice enhancement.  Maybe a `openEventsSent` array that contains the keys we've notified for?  That way a slot change doesn't blow away that info?

Reply:
Yeah either the keys or you could have a `Map` of the element nodes themselves. That way if the slot changed and they replaced the node with another empty one you'd fire the event again. But maybe I'm thinking too much into this and it's more complicated than it's worth. Could save this for if it actually comes up.

Reply:
Looking at current usages, there is one place that is using the repeat notification as a feature.  They display/hide a button in the footer based on being on that tab.  The new filter won't let them do this, so that will need to be solved a different way regardless.